src/avfoundation.cs

```diff
@@ -11350,6 +11844,10 @@ interface AVContentKeySession {
 		[Export ("renewExpiringResponseDataForContentKeyRequest:")]
 		void RenewExpiringResponseData (AVContentKeyRequest contentKeyRequest);
 
+		[NoWatch, NoTV, NoMac, iOS (11,0)]
+		[Export ("makeSecureTokenForExpirationDateOfPersistableContentKey:completionHandler:")]
+		void MakeSecureToken (NSData persistableContentKeyData, Action<NSData, NSError> handler);
```

Comment:
async methods should not return void
I think our intro tests special case (won't report) those - but otherwise will scream if we miss an Async